plexapi/library.py

```diff
@@ -1083,6 +1111,111 @@ def __len__(self):
         return self.size
 
 
+@utils.registerPlexObject
+class Station(PlexObject):
+    """ Represents the Station area in the MusicSection.
+
+        Attributes:
+            TITLE (str): 'Stations'
+            TYPE (str): 'station'
+            hubIdentifier (str): Unknown.
+            size (int): Number of items found.
+            title (str): Title of this Hub.
+            type (str): Type of items in the Hub.
+            more (str): Unknown.
+            style (str): Unknown
+            items (str): List of items in the Hub.
+    """
+    TITLE = 'Stations'
+    TYPE = 'station'
+
+    def _loadData(self, data):
+        """ Load attribute values from Plex XML response. """
+        self._data = data
+        self.hubIdentifier = data.attrib.get('hubIdentifier')
+        self.size = utils.cast(int, data.attrib.get('size'))
+        self.title = data.attrib.get('title')
+        self.type = data.attrib.get('type')
+        self.more = data.attrib.get('more')
+        self.style = data.attrib.get('style')
+        self.items = self.findItems(data)
+
+    def __len__(self):
+        return self.size
+
+
+class Sort(PlexObject):
+    """ Represents a
+
+    """
+    TAG = 'Sort'
+
+    def _loadData(self, data):
+        """ Load attribute values from Plex XML response. """
+        self._data = data
+        self.defaultDirection = data.attrib.get('defaultDirection')
+        self.descKey = data.attrib.get('descKey')
+        self.key = data.attrib.get('key')
+        self.title = data.attrib.get('title')
+        self.firstCharacterKey = data.attrib.get('firstCharacterKey')
+
+
+class Field(PlexObject):
```

Comment:
We already have a class called Field. Maybe change the name or atleast make a better docstring.

Reply:
oh yeah docstrings need updated. I based the name off the xml tagging. Technically they can both exist after 3746b4e . But, yeah, I can change it.